src/pages/guide/comparison-with-leading-frameworks.ejs

```diff
@@ -229,7 +229,7 @@ const data = {
       </tr>
       <tr>
         <td><b>VS Code extension</b></td>
-        <td><span class="comp-yes">✅</span> <a href="#">Try now</a></td>
+        <td><span class="comp-yes">✅</span> <a href="https://marketplace.visualstudio.com/items?itemName=browserstackcom.nightwatch#:~:text=The%20Nightwatch%20Extension%20for%20Visual,in%20a%20single%2C%20integrated%20environment.html">Try now</a></td>
```

Comment:
```suggestion
        <td><span class="comp-yes">✅</span> <a href="https://marketplace.visualstudio.com/items?itemName=browserstackcom.nightwatch">Try now</a></td>
```

Reply:
Actually, yesterday I tried below link but idk why it didn't work. So there is another place where we used above link. So I copied from there.
Now it is working, So changed there too.